srcpkgs/goffice/template

```diff
@@ -1,9 +1,9 @@
 # Template file for 'goffice'
 pkgname=goffice
 version=0.10.43
-revision=3
+revision=4
 hostmakedepends="automake pkg-config pcre gettext-devel intltool libtool glib-devel gtk-doc"
-makedepends="GConf-devel gtk+3-devel librsvg-devel libgsf-devel libxml2-devel pcre-devel libxslt-devel"
+makedepends="gtk+3-devel librsvg-devel libgsf-devel libxml2-devel pcre-devel libxslt-devel"
```

Comment:
Don't think it is necessary to revbump, or GConf-devel is still used ?

Reply:
I removed the revbump's when no change would result to the built package.